inspec.yml

```diff
@@ -0,0 +1,611 @@
+      - "SV-230248" # SI-11
+      - "SV-230249" # SI-11
+      - "SV-230250" # SI-11
+      - "SV-230276" # SI-16
+      - "SV-230280" # SI-16
+    <% else %>
+    required: true
+    value: []
+    <% end %>
+
+#SV-230244, SV-244525:
+  - name: mins_of_inactivity
+    desc: Maximum number of minutes of account inactivity before account lockout
+    type: Numeric
+    value: 30
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: system_inactivity_timeout
+    desc: Maximum system inactivity timeout (time in seconds).
+    type: Numeric
+    value: 1800
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230244
+  - name: client_alive_interval 
+    desc:
+    type: Numeric
+    value: 1800
+
+#SV-230331, SV-230374
+  - name: temp_account_expire_time
+    desc: Temporary (short term) should be set with an expiration date/time (time in hours)
+    type: Hash
+    value: 
+      hours: 24
+      days: 1
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230483, SV-244543
+  - name: storage_volume
+    desc: Amount of audit storage volume
+    type: Numeric
+    value: 80
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: storage_space_left
+    desc: Amount of audit storage volume left
+    type: Numeric
+    value: 20
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230332, SV-230333, SV-230334, SV-230335, SV-230336, SV-230337, SV-230338, SV-230340, SV-230341, SV-230344, SV-230345
+  - name: unsuccessful_attempts
+    desc: Maximum number of unsuccessful attempts before lockout
+    profile: redhat-enterprise-linux-8-stig-baseline
+    type: Numeric
+    <% if ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'] == 'Moderate-HVA' || ENV['BASELINE'] == 'Low' || ENV['BASELINE'] == 'Low-HVA' || ENV['BASELINE'].nil? %>
+    value: 5
+    <% else %>
+    value: 3
+    <% end %>
+
+  - name: fail_interval # also in SV-230535 
+    desc: Interval of time in which the consecutive failed logon attempts must occur in order for the account to be locked out (time in seconds)
+    type: Numeric
+    value: 7200
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: fail_interval_mins # also in SV-230535
+    desc: Interval of time in which the consecutive failed logon attempts must occur in order for the account to be locked out (time in minutes)
+    type: Numeric
+    value: 120
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: unlock_time
+    desc: Amount of time it will take for an account to be unlocked
+    type: Numeric
+    value: 3600
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: lockout_time
+    desc: Minimum amount of time account must be locked out after failed logins. This attribute should never be set greater than 604800 (time in seconds).
+    type: Numeric
+     <% if ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'] == 'Moderate-HVA' || ENV['BASELINE'].nil? %>
+    value: 3600
+    <% else %>
+    value: 900
+    <% end %>
```

Comment:
Low condition should be listed separately (to avoid confusion) and High needs to be set to `0` (also add `# same as baseline` for clarity)